packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -58,35 +56,40 @@ const ChartConfigTabs: FC = () => {
     const items = [...dimensionsInMetricQuery, ...metricsAndTableCalculations];
 
     return (
-        <Box w={320}>
-            <Tabs
-                onChange={setTab}
-                selectedTabId={tab}
-                renderActiveTabPanelOnly
-            >
-                <Tab
-                    id="layout"
-                    title="Layout"
-                    panel={<FieldLayoutOptions items={items} />}
-                />
-                <Tab
-                    id="series"
-                    title="Series"
-                    panel={<SeriesTab items={items} />}
-                />
-                <Tab
-                    id="axes"
-                    title="Axes"
-                    panel={<AxesOptions items={items} />}
-                />
-                <Tab
-                    id="legend"
-                    title="Display"
-                    panel={<LegendPanel items={items} />}
-                />
-                <Tab id="grid" title="Margins" panel={<GridPanel />} />
-            </Tabs>
-        </Box>
+        <Tabs w={335} defaultValue="layout">
```

Comment:
will w="100%" work here?

Reply:
No, this one is slightly wider than all of the others. It has 5 tabs. In my draggable prototype I set these up to all share the same popover, but for now they all make their own and set up the width themselves.